cmd/orb.go

```diff
@@ -41,10 +41,10 @@ func newOrbCommand() *cobra.Command {
 
 	orbCommand.AddCommand(orbListCommand)
 
-	orbValidateCommand.PersistentFlags().StringVarP(&orbPath, "path", "p", "orb.yml", "path to orb file")
+	orbValidateCommand.PersistentFlags().StringVarP(&orbPath, "path", "p", "orb.yml", "path to orb file ('-' for STDIN)")
```

Comment:
Having to type `-p` when I just want to pipe some file as stdin to a program is painful.

For certain flags, which aren't optional (behavior changing) such as the path to your config, a truly vital option for this config (or we default) shouldn't be a flag IMO.

I'd like if we could just type `circleci orb validate path/to/orb.yml` or leave `path/to/orb.yml` out entirely for the same results (if it were default) and in this use-case `circleci orb validate -`. But I may just be dreaming. :thinking:

Reply:
+1 - my understanding is we added path as a flag because we would have a default value of the path, but we should probably kill the default value, have path be required, pass it without a flag. I think we talked about a pattern where we'd try to limit required arguments and let you pass them without a flag. But if that's a major effort we should discuss its relative value.

Reply:
Let's not make the config path be required. The current CLI tool has a command `circleci config validate` and we don't want to break our customer's workflows.